tests/prism/test_pipelines.py

```diff
@@ -140,13 +140,7 @@ def test_RNAseq_pipeline_config_generation_after_prismify(prismify_result, templ
         return
 
     full_ct = get_test_trial(
-        [
-            "CTTTPP111.00",
-            "CTTTPP121.00",
-            "CTTTPP122.00",
-            "CTTTPP121.00",
-            "CTTTPP123.00",
-        ],
+        ["CTTTPP111.00", "CTTTPP121.00", "CTTTPP122.00", "CTTTPP123.00"],
```

Comment:
```suggestion
        ["CTTTPP111.00", "CTTTPP121.00", "CTTTPP211.00", "CTTTPP221.00"],
```
so we have multiple participants in this test example, maybe?